Anne-Hyacinthe/short-response.md

```diff
@@ -0,0 +1,68 @@
+# Problem Set 2.2 - Variables and Control Flow
+## Short Response Questions
+
+1. **What does the code below log? Why?**
+   ```javascript
+    {
+      let singer = 'Omar Apollo';
+    }
+
+    console.log(`My favorite singer is ${singer});
+   ```
+   * It logs an error because singer is defiend in a scope but is being called outside of that scope.
```

Comment:
Yes! Small language improvement:
> It logs an error because `singer` is defined in a **block** scope but is being ~~called~~ referenced outside of that scope.

You _call_ functions. You _reference_ variables.